### PR TITLE
[anxpro] de-activated all fiat currencies (no deposits or withdrawals are allowed).  See: https://anxpro.com/pages/fees

### PR DESCRIPTION
**Pull request by npomfret**

### CODE REVIEW

**Comment by kroitor** [comment]
@npomfret hm.... I can find any info about it on the internet... can you? Is that permanent or temporary?

**Comment by npomfret** [comment]
It's been like that for a long time now, like several months.  The FIAT markets are still active but there's obviously not much volume: https://snowmonkey.co.uk/exchange/anxpro (and I think their tickers are reporting incorrect volumes as loads of markets have exactly the same number).

Anyway, you can see on their [fees](https://anxpro.com/pages/fees) page there are no deposits for "All Fiat Currencies", and the only FIAT withdrawals are for HKD.

We don't have the ability to model deposits and withdrawals being active separately, so I marked all of them (including HKD) as inactive.

**Comment by npomfret** [comment]
@kroitor i don't understand the error in the build- would you mind taking a look?

**Comment by npomfret** [comment]
@kroitor I've removed all the hardcoded markets and currencies now (using a new endpoint the v3 api)

File: js/anxpro.js [hunk 1]
```diff
@@ -23,14 +22,23 @@ module.exports = class anxpro extends Exchange {
...
                 'www': 'https://anxpro.com',
                 'doc': [
                     'https://anxv2.docs.apiary.io',
                     'https://anxpro.com/pages/api',
                 ],
             },
             'api': {
+                'v3public': {
+                    'get': [
+                        'currencyStatic',
```
Comment (kroitor) [inline js/anxpro.js]: > @kroitor I've removed all the hardcoded markets and currencies now (using a new endpoint the v3 api)

Awesome! Much better than it was before! ) One question, though: we can't list just one endpoint if we are adding an API section – can you list the rest of them (public) here?

Comment (npomfret) [reply js/anxpro.js]: The v3 api literally only has one public endpoint.

It does have a load of private API calls, but I've not implemented those.

Comment (kroitor) [reply js/anxpro.js]: @npomfret got it, thx for clarifying this and for the effort! Appreciate your involvement! Merging shortly!

**Comment by npomfret** [comment]
Oops, I reset my branch (thinking it had been merged).  Have moved the change to a new pr https://github.com/ccxt/ccxt/pull/4645

**Pull request closed without merging**